prod/mach_zehnder_gadget/javascript/app/WebSocketController.js

```diff
@@ -1,5 +1,4 @@
 myApp.controller('WebSocketController',['$scope', function($scope) {
```

Comment:
in general in this file, you are not consistent with leaving or omitting empty lines. Try to be consistent. I prefer to omit the empty lines unless it's a new function coming.